source/core/index.ts

```diff
@@ -1350,14 +1383,33 @@ export default class Request extends Duplex implements RequestEvents<Request> {
 		delete options.request;
 		delete options.timeout;
 
-		const requestOptions = options as unknown as https.RequestOptions;
+		const requestOptions = options as unknown as RealRequestOptions;
 
-		if (options.certificateAuthority) {
-			requestOptions.ca = options.certificateAuthority;
-		}
+		// HTTPS options remapping
+		if (options.https) {
+			if ('rejectUnauthorized' in options.https) {
```

Comment:
```suggestion
			if (options.https.rejectUnauthorized) {
```

Reply:
I wrote that line like that because
`if(options.https.rejectUnauthorized)` is not taken if `https.rejectUnauthorized` is `false`, so `https.rejectUnauthorized` is unable to override `rejectUnauthorized` if `false`.